core/src/i18n/en.yaml

```
@@ -167,6 +167,24 @@ deployments:
   title: Deployments
 discovery-and-network:
   title: Discovery and Network
+dnsentries:
+  headers:
+    dns-name: DNS Name
+    provider: Provider
+    provider-type: Provider type
```

Not sure what it looks like on the UI, but we might want to consider capitalization - see [English: General Rule for Capitalization (Sentence Style and Title Case) on the User Interface](https://help.sap.com/viewer/DRAFT/e33c591ae4494a659a3f5f983c9d1161/PROD/en-US/79ce282052d14cc38e6a203a1fc277c4.html).

```suggestion
    provider-type: Provider Type
```